modules/openy_features/openy_media/modules/openy_media_image/config/install/core.entity_view_display.media.image.icon.yml

```diff
@@ -9,7 +9,7 @@ dependencies:
     - field.field.media.image.field_media_tags
     - media.type.image
   module:
-    - blazy
```

Comment:
@jediandrey please check what is happening here

Reply:
resolved?